packages/web-ui/src/ds/molecules/DocumentTextEditor/index.tsx

```diff
@@ -25,7 +26,15 @@ function EditorWrapper(props: DocumentTextEditorProps) {
   // When imported, Monaco automatically tries to use the window object.
   // Since this is not available when rendering on the server, we only
   // render the fallback component for SSR.
-  if (typeof window === 'undefined') return <DocumentTextEditorFallback />
+  const [isClient, setIsClient] = useState(false)
+
+  useEffect(() => {
+    setIsClient(typeof window !== 'undefined')
+  }, [])
+
+  if (!isClient) {
+    return <DocumentTextEditorFallback />
+  }
   return <DocumentTextEditor {...props} />
```

Comment:
This fixes the "hydration" error that Next.js shows when loading the editor, because it rendered something different in SSR and in the browser. Now, both render the fallback at first and then the browser updates to the actual editor (which renders the fallback too until monaco fully loads anyways)